### ISSUE TITLE
Supply variables to the jsonExport scipt: 

### ISSUE THREAD

**Issue by leonprou** [liquify-validation/Fuse_spark_test_script#4]
- skipPieCharts (instead of the default now - 80)
-  add an flag to create pieChards or show the graph of balances

**Comment by Andrew-Pohl** [comment]
Yep sensible, I will also make it save the graphs too (like it does with the pie charts 👍)
